students/mark_luckeroth/lesson03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+Names = ['Peter','Paul','Mary','Jake','Raj']
+Donations = [[10., 10., 10.],
+            [5., 50000., 5., 5.],
+            [100.],
+            [123., 456., 789.],
+            [60., 600000.]]
```

Comment:
per pep8 var names should use lower case snake_case -> `names`, `donations`

also, an alternate data structure:
```
donors = [('Peter', [10, 20, 30]), ('Paul', [29])]
```
it usually simplifies the code